css/index.css

```diff
@@ -101,17 +101,45 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
```

Comment:
The .boxes is the parent class to .box1 .box2 etc.  It will automatically inherit all of the parent classes styling so you can omit all of the specific box class names from here.